jkube-kit/jkube-kit-wildfly-jar/src/test/java/org/eclipse/jkube/wildfly/jar/enricher/WildflyJARHealthCheckEnricherTest.java

```diff
@@ -99,22 +107,56 @@ public void testCloudConfiguration(@Mocked final JavaProject project) {
         assertEquals("/health/live", probe.getHttpGet().getPath());
         assertEquals("HTTP", probe.getHttpGet().getScheme());
         probe = enricher.getReadinessProbe();
+        assertNotNull(probe);
         assertEquals(9990, probe.getHttpGet().getPort().getIntVal().intValue());
         assertEquals("/health/ready", probe.getHttpGet().getPath());
         assertEquals("HTTP", probe.getHttpGet().getScheme());
+        probe = enricher.getStartupProbe();
+        assertNull(probe);
+    }
+
+    @Test
+    public void testCloudConfiguration_withWildflyJarAfter25_0shouldAdd_startupProbe() {
+        // Wildfly version must be at least 25.0.0.Final to add startup probe
+        wildFlyJarDependencyWithVersion("26.1.1.Final");
+
+        Map<String, Object> config = new HashMap<>();
+        config.put("cloud", null);
+        setupExpectations(project, config, Collections.emptyMap());
+        WildflyJARHealthCheckEnricher enricher = new WildflyJARHealthCheckEnricher(context);
+        Probe probe = enricher.getLivenessProbe();
         assertNotNull(probe);
+        assertEquals(9990, probe.getHttpGet().getPort().getIntVal().intValue());
+        assertEquals("/health/live", probe.getHttpGet().getPath());
+        assertEquals("HTTP", probe.getHttpGet().getScheme());
+        probe = enricher.getReadinessProbe();
+        assertNotNull(probe);
+        assertEquals(9990, probe.getHttpGet().getPort().getIntVal().intValue());
+        assertEquals("/health/ready", probe.getHttpGet().getPath());
+        assertEquals("HTTP", probe.getHttpGet().getScheme());
+        probe = enricher.getStartupProbe();
+        assertNotNull(probe);
+        assertEquals(9990, probe.getHttpGet().getPort().getIntVal().intValue());
+        assertEquals("/health/started", probe.getHttpGet().getPath());
+        assertEquals("HTTP", probe.getHttpGet().getScheme());
     }
 
     @Test
-    public void testWithCustomConfigurationComingFromConf(@Mocked final JavaProject project) {
+    public void testWithCustomConfigurationComingFromConf_withWildflyJarBefore25_0shouldNotAdd_startupProbe() {
+        wildFlyJarDependencyWithVersion("24.1.1.Final");
+
         Map<String, Object> jarConfig = new HashMap<>();
         jarConfig.put("cloud", null);
         final Map<String, Map<String, Object>> config = createFakeConfig("{\"readinessPath\":\"/foo/ready\","
                 + "\"livenessPath\":\"/foo/live\","
                 + "\"port\":\"1234\","
-                + "\"scheme\":\"https\","
+                + "\"scheme\":\"HTTPS\","
```

Comment:
You shouldn't be changing existing test expectations. It was failing because we missed `toUpperCase` while parsing scheme